### ISSUE TITLE
Adding Apple Silicon(Arm) Client for mac

### ISSUE THREAD

**Issue by ABDERRAHMANE-OUALI** [kaiserdj/Darkorbit-client#62]
Is there a plan to release the arm version of the client for mac ?

**Comment by kaiserdj** [comment]
I have to investigate to add direct support, but for now you can use the client.

Check this link where I explain how you can do it

https://github.com/kaiserdj/Darkorbit-client/issues/58#issuecomment-1150071047

**Comment by kaiserdj** [comment]
Could you try this version and tell me if it works well for you on a mac m1 without modifying anything:
https://github.com/Rolworld/Darkorbit-client-test/releases/download/v2.7.4/DarkOrbit-Client-2.7.4-arm64.dmg

I don't have a mac, so I can't test it

**Comment by ABDERRAHMANE-OUALI** [comment]
it says File is damaged and cannot be opened after I try to open the app

**Comment by kaiserdj** [comment]
Does the same thing happen to you with this one?
https://github.com/Rolworld/Darkorbit-client-test/releases/download/v2.7.5/DarkOrbit-Client-2.7.5-arm64.dmg

**Comment by ABDERRAHMANE-OUALI** [comment]
I test it but still gives me this the same error of damage file

**Comment by kaiserdj** [comment]
> I have to investigate to add direct support, but for now you can use the client.
> 
> Check this link where I explain how you can do it
> 
> [#58 (comment)](https://github.com/kaiserdj/Darkorbit-client/issues/58#issuecomment-1150071047)

Did you try what it tells you?

**Comment by ABDERRAHMANE-OUALI** [comment]
for some reason even when I run the intel version the game isn't showing but when I go activity monitor the game is launched , and also I didn't find run with Rosita for some reason
<img width="270" alt="Screen Shot 2022-08-17 at 08 50 15" src="https://user-images.githubusercontent.com/60064144/185064770-5f96e8e6-a84a-4bbe-8d10-c1913005f907.png">

**Comment by kaiserdj** [comment]
In the comment you should have the solution:
https://github.com/kaiserdj/Darkorbit-client/issues/58#issuecomment-1158473223

**Comment by ABDERRAHMANE-OUALI** [comment]
Open Using Rosetta options is only show to Universal executable not Intell ones

**Comment by kaiserdj** [comment]
What you said doesn't make any sense...
Rosetta enables Mac computers with Apple chips to use apps designed for Mac computers with Intel processors.
If not, what is the logic of using rosetta????

**Comment by ABDERRAHMANE-OUALI** [comment]
I do have Rosetta install but the option "Open With Rosetta" isn't showing

**Comment by kaiserdj** [comment]
@alexperez52 didn't get it either (https://github.com/kaiserdj/Darkorbit-client/issues/58#issuecomment-1158473223), but he put what he did to make it work for him. Did you try with that?

**Comment by kaiserdj** [comment]
> for some reason even when I run the intel version the game isn't showing but when I go activity monitor the game is launched , and also I didn't find run with Rosita for some reason <img alt="Screen Shot 2022-08-17 at 08 50 15" width="270" src="https://user-images.githubusercontent.com/60064144/185064770-5f96e8e6-a84a-4bbe-8d10-c1913005f907.png">

Btw, this file is the installer. Have you installed the client?

**Comment by ABDERRAHMANE-OUALI** [comment]
yes I installed the client but when I open it, it shows nothing 

<img width="1440" alt="Screen Shot 2022-08-18 at 16 02 21" src="https://user-images.githubusercontent.com/60064144/185428760-6df483c6-fcc4-4d47-b98e-8c61b26b9065.png">


in task manager it show the client is running but nothing is shown

**Comment by kaiserdj** [comment]
To clear all launcher data, delete the following folder:
Mac: ~/Library/Application Support/darkorbit-client
Close the launcher completely and delete the folder indicated above, if it continues to give problems let me know

**Comment by ABDERRAHMANE-OUALI** [comment]
I did delete the darkorbit client app and re-install it but the problem
persists

**Comment by v3n7i** [comment]
Can't enter the game as well with Mac M1. It cannot load PepperFlashPlayer (fat file, but missing compatible architecture (have (i386,x86_64), need (arm64e))).
I don't want to install other versions of MacOS or to mess up with the OS. Better fix this issue out of the box instead of using workarounds, more people will encounter the issue with newer chips.

**Comment by kaiserdj** [comment]
> Can't enter the game as well with Mac M1. It cannot load PepperFlashPlayer (fat file, but missing compatible architecture (have (i386,x86_64), need (arm64e))). I don't want to install other versions of MacOS or to mess up with the OS. Better fix this issue out of the box instead of using workarounds, more people will encounter the issue with newer chips.

The problem is that flash player stopped being developed before the release of the m1 processors (arm architecture) so a specific version was never developed for this mac model.
I will investigate more on the matter, but it is difficult to do tests without having an m1

**Comment by emrementese** [comment]
I used long time (5-6 month) kaiserdj's clients with m1 mac air.  I Haven't got a problem.

**Comment by BenjaminEmiliani** [comment]
For those that do see the checkbox “Open with Rosetta” I suggest updating to Rosetta 2 and then simply double clicking the mac executable obtained by downloading this release:
https://github.com/Rolworld/Darkorbit-client-test/releases/download/v2.7.4/DarkOrbit-Client-2.7.4-arm64.dmg

I don’t see the checkbox either but by doing the above, I no longer have the issue
